lib/bumper.js

```diff
@@ -66,10 +66,7 @@ class Bumper {
    */
   getOpenPrScope () {
     const vcs = this.vcs
-    return utils.getSha(this.config, vcs)
-      .then((sha) => {
-        return vcs.getOpenPrForSha(sha)
```

Comment:
I was able to factor out both the `getSha()` and the `getOpenPrForSha()` methods because the CI environments we support both have access to the pull request number, which is a much more efficient way to get the PR from the API than by looking for all PRs that match a sha hash.